src/Sulu/Bundle/AdminBundle/FormMetadata/FormXmlLoader.php

```diff
@@ -69,6 +70,10 @@ protected function parse($resource, \DOMXPath $xpath, $type): LocalizedFormMetad
         // init running vars
         $tags = [];
 
+        if ($xpath->query('/x:form')->count() === 0) {
+            throw new InvalidRootTagException($resource, 'form');
```

Comment:
Maybe we could simply even skip that file, but not sure if this will hide some bugs, that would be hard to find then :thinking: